lab06-kaylee/test/tcp-test.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const server = require('../server.js');
+const expect = require('chai').expect;
+const net = require('net');
+//Not sure if I need to require anything else in... TBD.
```

Comment:
**TODO FOR LOST POINTS**
The answer is no, but remember that we're getting strict on comments now.